File: src/frontend/src/app/model/project-fields/ListElementsContainerDto.ts
import { ListElementDto } from './ListElementDto';
import { ProjectFieldDto } from './ProjectFieldDto';

export class ListElementsContainerDto extends ProjectFieldDto {
    public elements: Array<ListElementDto> = [];

    constructor(id: number, fieldType: string, fieldName: string, isRequired: boolean, elements: Array<ListElementDto>) {
        super(id, fieldType, fieldName, isRequired);
        this.elements = elements;
    }
}

File: src/main/java/scrumweb/dto/projectfield/ListElementsContainerDto.java
package scrumweb.dto.projectfield;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Set;

@NoArgsConstructor
@Getter @Setter
public class ListElementsContainerDto extends ProjectFieldDto {
    private Set<ListElementDto> elements;

    public ListElementsContainerDto(Long id, String fieldType, String fieldName, Boolean isRequired, Set<ListElementDto> elements) {
        super(id, fieldType, fieldName, isRequired);
        this.elements = elements;
    }
}
